moduleroot/spec/acceptance/nodesets/docker/debian-8.yml

```diff
@@ -0,0 +1,11 @@
+HOSTS:
+  debian-8-x64:
+    platform: debian-8-amd64
+    hypervisor: docker
+    image: debian:8
+    docker_preserve_image: true
+    docker_cmd: '["/sbin/init"]'
+    docker_image_commands:
+      - 'apt-get update && apt-get install -y net-tools wget locales strace lsof && echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && locale-gen'
+CONFIG:
+  trace_limit: 200
```

Comment:
Yeah, those docker nodesets are used on travis. At one point we might want to re-use them to prove that facts and plugins do not kill agents on unsupported platforms (or when dependencies are missing).